src/test/scala/io/getclump/Spec.scala

```diff
@@ -1,11 +1,23 @@
 package io.getclump
 
-import org.specs2.mock.Mockito
-import org.specs2.mutable.Specification
-import org.specs2.time.NoTimeConversions
+import utest._
+import scala.reflect.ClassTag
 
-trait Spec extends Specification with Mockito with NoTimeConversions {
+trait Spec extends TestSuite {
 
-  protected def clumpResult[T](clump: Clump[T]) =
-    awaitResult(clump.get)
+  object NoError extends Exception
+
+  protected def assertFailure[T: ClassTag](f: Future[_]) =
+    f.map(_ => throw NoError).recover {
+      case NoError => throw new IllegalStateException("A failure was expected.")
+      case e: T    => None
+    }
```

Comment:
I'll prepare a pull request to `utest` to introduce support for `Future` to `intercept`. This is a temporary solution.